src/components/search/GlobalSearchField.js

```diff
@@ -580,7 +580,7 @@ function GlobalSearchField(props) {
             setTeamSearchQueryEnabled(false);
             setOptions([]);
         }
-        setSearchTerm(value);
+        setSearchTerm(value.trim());
```

Comment:
This works mostly but you might want to cover the case where `value` might be `null` or `undefined` ?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining might help.

Reply:
Thank you, I'll remember to use it for future functions